src/Billing/BillingDetails/BillingDetails.js

```diff
@@ -0,0 +1,363 @@
+import React, {useEffect, useState} from 'react';
+import Card from "./Card/Card";
+import s from './BillingDetails.module.css'
+import {Modal, FormControl, Form, Col} from "react-bootstrap";
+import axios from 'axios'
+
+const cscRegEx = /\b\d{3}\b/;
+const cardRegEx = /\b\d{16}\b/;
+
+
+const url = "localhost:8041";
+
+const propss = {
+    data: [
+        {
+            'id': 1,
+            'cardNumber': '1234567898765432',
+            'cardName': 'Ivan Ivanov',
+            'expirationMonth': '05',
+            'expirationYear': '22',
+            'billingAddress': 'Pushkina dom kolotushkina'
+        },
+        {
+            'id': 2,
+            'cardNumber': '1234567898765432',
+            'cardName': 'kum',
+            'expirationMonth': '15',
+            'expirationYear': '23',
+            'billingAddress': 'Pushkina dom ssdssds'
+        },
+        {
+            'id': 3,
+            'cardNumber': '1234567898765432',
+            'cardName': 'kudsdsdm',
+            'expirationMonth': '15',
+            'expirationYear': '23',
+            'billingAddress': 'Pushkina dom ssdssds'
+        }
+    ]
```

Comment:
Do you need this extra data in your component?

Reply:
No, sorry. I will change)